charmhub/find.go

```diff
@@ -39,7 +39,14 @@ func (c *FindClient) Find(ctx context.Context, query string) ([]transport.FindRe
 		return nil, errors.Trace(err)
 	}
 
-	path, err = path.Query("fields", defaultFindFilter())
+	var fields string
+	if query != "" {
+		fields = defaultFindFilter()
+	} else {
+		fields = subsetFindFilter()
+	}
```

Comment:
Let's go for it and only request the Find data we use: names, bundle, version, publisher and summary?  We do not print anything different based on query vs all.